### PR TITLE
Use official Conan packages

### PR DESCRIPTION
**Pull request by uilianries**
Hi!

This PR brings just some enhancements:

- Download both Boost and Conan from official maintainers
- Apply C++11 support by property on CMake

Regards!

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,6 +8,6 @@ else()
 endif()
 conan_basic_setup()
 
-add_compile_options(-std=c++11)
```
Comment (memsharded) [inline CMakeLists.txt]: If this worked in cmake 2.8.12, I wouldn't update it. 
There are still some conan users that are using cmake 2.8.12, so it is good to try to keep as compatible with them as possible.

**Comment by uilianries** [comment]
Agreed!

But what about to just use this flag for gcc and clang? This example could
be executed with visual studio.

**Comment by memsharded** [comment]
I typically work in Windows and VS :). It is just a warning for that case, it doesn't error.
But sure, it could be guarded with some ``if()``.

**Comment by uilianries** [comment]
Let's keep cmake file as before. It does work with msvc so I see no problem.

**Comment by memsharded** [comment]
Thanks!!

**Pull request merged**